src/test/regress/expected/with_basics.out

```diff
@@ -0,0 +1,274 @@
+-- CTE cannot be outside of FROM/WHERE clause
```

Comment:
The comment is not accurate. We're supporting the query, it just needs to return single row